packages/@aws-cdk/aws-lambda-python/README.md

```diff
@@ -172,4 +199,33 @@ new lambda.PythonFunction(this, 'function', {
 });
 ```
 
-This type of an example should work for `pip` and `poetry` based dependencies, but will not work for `pipenv`.
+**Note:** Setting custom build args for bundling will force the base bundling image to be rebuilt every time (i.e. skip the Docker cache).
+
+Setting only environment variable should work for `pip` and `poetry` based dependencies, whereas `pipenv` based dependencies will require **both** build args and environment variables to be set.
+
+
+Example for using Code Artifact with `pipenv`-based dependencies:
```

Comment:
Running into this also made clear how we could use `pipenv` with Code Artifact.

Reply:
Why does `pipenv` require both env vars and build args?

Reply:
Thanks for asking, @corymhall. That was an incorrect note on my part stemming from how bundling was occurring earlier (in the Docker build step), and I didn't completely understand it since I don't use `pipenv`.

Over the last couple days, I created a test project with `pipenv` and validated installing a package into it that is only on Test PyPI. It works when the `PIP_INDEX_URL` is set, which it'll be when the Docker run step occurs. I updated the documentation now to reflect that.